packages/c/clmtrackr.json

```
@@ -14,11 +14,11 @@
     "url": "https://github.com/auduno/clmtrackr.git"
   },
   "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/auduno/clmtrackr.git",
+    "source": "npm",
+    "target": "clmtrackr",
```

```suggestion
    "source": "git",
    "target": "git://github.com/auduno/clmtrackr.git",
```
The [git repo has the versions tagged correctly](https://github.com/auduno/clmtrackr/tree/v1.1.2/build), let's not complicate the change by updating where we get the files from.